robottelo/cli/hostcollection.py

```diff
@@ -50,6 +50,11 @@ def transform_ids(cls, options=None):
             ]
             if host_ids:
                 options['host-ids'] = ','.join(host_ids)
+        if 'host-id' in options:
+            host_id = options.get('host-id')
+            if host_id and not host_id.isdigit():
+                host_name = ContentHost.info({'id': host_id})['name'].lower()
+                options['host-id'] = Host.info({'name': host_name})['id']
```

Comment:
think this function should be refactored to check options, after forget to check every time
at the top of this function 
```python
if options is None:
    # do nothing
    return
```
and invoke it without checking at all 
```python
 cls.transform_ids(options)
```